src/System.Security.Cryptography.X509Certificates/tests/FindTests.cs

```diff
@@ -84,9 +86,13 @@ private static void RunSingleMatchTest_PfxCer(X509FindType findType, object find
                 Assert.Equal(expected, match);
                 Assert.NotSame(expected, match);
 
-                // Verify that the find result and original are linked, not just equal.
-                match.FriendlyName = "HAHA";
-                Assert.Equal("HAHA", expected.FriendlyName);
+                // FriendlyName is Windows-only.
```

Comment:
Nope, FriendlyName and Archived are the only mutatable properties on Windows, and there aren't any on non-Windows.  So there's not really a notion of being linked that needs to be preserved on Unix.